src/components/pages/use-cases/use-case-card/use-case-card.jsx

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+import ArrowIcon from 'icons/arrow.inline.svg';
+
+import quotes from './images/quotes.svg';
+
+const UseCaseCard = ({ heading, testimonials }) => (
+  <section className="bg-gray-4">
+    <Container className="flex flex-col items-center py-10 md:py-20 lg:py-28">
+      <Heading
+        className="mb-8 max-w-full text-center leading-tight lg:mb-12 lg:max-w-[70%] lg:leading-tight xl:leading-tight"
+        tag="h2"
+        size="md"
+      >
+        {heading}
+      </Heading>
+      <ul className="flex flex-col gap-y-8 md:gap-y-10">
+        {testimonials.map(({ logo, url, CTAtext, description, title, quote, person }, index) => {
+          const Logo = logo;
+          return (
+            <li
+              className="flex flex-col gap-0 rounded-lg bg-white px-6 py-8 shadow-primary sm:items-center md:flex-row md:gap-4 md:px-12"
+              key={title + index}
+            >
+              <div className="basis-2/4 items-center justify-around gap-8 md:flex">
+                <div className="flex flex-col items-center gap-6">
+                  <Logo />
+                  {url && (
+                    <a
+                      href={url}
+                      target="_blank"
```

Comment:
The first url is linked to cilium blog, I suppose we don't need to target blank in this case